setup.sh

```diff
@@ -169,12 +169,20 @@ ${ORANGE}OPTIONS${RESET}
 
 ${RED}[${ORANGE}SUB${RED}]${ORANGE}COMMANDS${RESET}
     ${LBLUE}COMMAND${RESET} email ${RED}:=${RESET}
-        ${0} email ${CYAN}add${RESET} <EMAIL ADDRESS> [<PASSWORD>]
-        ${0} email ${CYAN}update${RESET} <EMAIL ADDRESS> [<PASSWORD>]
-        ${0} email ${CYAN}del${RESET} [ OPTIONS${RED}...${RESET} ] <EMAIL ADDRESS> [ <EMAIL ADDRESS>${RED}...${RESET} ]
+        ${0} email ${CYAN}add${RESET} [-g] <EMAIL ADDRESS> [<PASSWORD>]
+        ${0} email ${CYAN}update${RESET} [-g] [-c] <EMAIL ADDRESS> [<PASSWORD>] [<OLD CRYPTOKEY PASSWORD>]
+        ${0} email ${CYAN}del${RESET} OPTIONS${RED}...${RESET} ] <EMAIL ADDRESS> [ <EMAIL ADDRESS>${RED}...${RESET} ]
```

Comment:
Was removing the bracket done on purpose?

Reply:
Whoops, didn't notice that. Thanks for pointing it out!